dmoj/cptbox/_cptbox.pyx

```diff
@@ -202,7 +211,7 @@ cdef class Debugger:
             # When using seccomp, -1 as syscall means "skip"; when we are not,
             # we swap with a harmless syscall without side-effects (getpid).
             if not PTBOX_SECCOMP and value == -1:
```

Comment:
Compile-time `PTBOX_SECCOMP`, not `use_seccomp`?

Reply:
Fixed.